packages/yoastseo/spec/languageProcessing/helpers/sentence/getSentencesSpec.js

```diff
@@ -523,9 +523,29 @@ describe( "a test for when texts containing sentence delimiter", () => {
 	} );
 
 	it( "can deal with the edge case of quotes around initials.", ()=>{
-		expect( getSentences( "The reprint was favourably reviewed by \"A. B.\" in The Musical Times in 1935, who commented \"Praise is due to Mr Mercer." ) ).toEqual(
+		const text = "The reprint was favourably reviewed by \"A. B.\" in The Musical Times in 1935, who commented \"Praise is due to Mr Mercer.";
+		expect( getSentences( text ) ).toEqual(
 			[ "The reprint was favourably reviewed by \"A. B.\" in The Musical Times in 1935, who commented \"Praise is due to Mr Mercer." ] );
 	} );
+
+	it( "should not break when the text starts with double quotation mark followed by space", ()=>{
+		let text = "\" This is a text with double quotation mark.";
+		expect( getSentences( text ) ).toEqual( [ "\"", "This is a text with double quotation mark." ] );
+		text = "\" \"This is a text with double quotation mark.";
```

Comment:
I had to read this twice. Nothing to block merging. But the fact that it is a double 'double quotation mark' is rather confusing.

Reply:
The purpose of the test with `"\" \"This is a text with double quotation mark."` is to make sure that the solution still words even when there are multiple double quotation marks. I'll adjust the testing text for that cases so it's less confusing :)